contrib/functions/go/generate-blueprint-docs/README.md

```diff
@@ -0,0 +1,164 @@
+# generate-blueprint-docs
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `generate-blueprint-docs` function generates opinionated documentation for a [blueprint](https://github.com/GoogleCloudPlatform/blueprints).
+
+It works by analyzing a blueprint kpt package to generate markdown documentation and writes it to a file.
+
+<!--mdtogo-->
+
+<!--mdtogo:Long-->
+
+## Usage
+
+The `generate-blueprint-docs` function is expected to be executed imperatively like:
+
+```shell
+kpt fn eval -i gcr.io/kpt-fn-contrib/generate-blueprint-docs:unstable --include-meta-resources \
+--mount type=bind,src="$(pwd)",dst="/tmp",rw=true
+```
+
+## FunctionConfig
+
+This function supports `ConfigMap` `functionConfig`.
```

Comment:
Can you please add the support for using a client-side CRD as `functionConfig`?
For new functions, we want to ensure it supports at least one client-side CRD and it can also support `ConfigMap` if it want to.
Client-side CRD will have schema and it will help validate the functionConfig. We can't do much for ConfigMap since it's unstructured.

Reply:
@mengqiy Currently I would expect most of the primary usage would be via imperative execution and fn args. Can we add this in the future if there are relevant user journeys?